### PR TITLE
NEWS for chop and chomp change

### PR DESCRIPTION
**Pull request by TotalVerb**
NEWS for #18339, cc @KristofferC

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Language changes
     This can be changed back to the original color by setting the environment variable `JULIA_INFO_COLOR` to `"blue"`.
     One way of doing this is by adding `ENV["JULIA_INFO_COLOR"] = :blue` to the `.juliarc.jl` file.
     For more information regarding customizing colors in the REPL, see this [manual section]( http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors).
+  * The `chop` and `chomp` functions new return `SubString`s.
```
Comment (KristofferC) [inline NEWS.md]: new -> now?

**Review by KristofferC — APPROVED** [review]

File: NEWS.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Language changes
     This can be changed back to the original color by setting the environment variable `JULIA_INFO_COLOR` to `"blue"`.
     One way of doing this is by adding `ENV["JULIA_INFO_COLOR"] = :blue` to the `.juliarc.jl` file.
     For more information regarding customizing colors in the REPL, see this [manual section]( http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors).
+  * The `chop` and `chomp` functions now return `SubString`s.
```
Comment (tkelman) [inline NEWS.md]: add a reference to the PR

Comment (stevengj) [inline NEWS.md]: i.e.
```
now return `SubString` ([#18339]).
```
and then re-run `doc/NEWS-update.jl` to update the links.

Comment (TotalVerb) [inline NEWS.md]: Thanks, fixed.

**Pull request merged**